spec/requests/api/merge_requests_spec.rb

```diff
@@ -114,6 +114,19 @@
     end
   end
 
+  describe "GET /projects/:id/merge_request/:merge_request_id/files" do
+    it "should return the file information of the merge_request" do
+      get api("/projects/#{project.id}/merge_request/#{merge_request.id}/files", user)
+      expect(response.status).to eq 200
+      expect(json_response['files'].size).to eq(merge_request.diffs.size)
+    end
+
+    it "returns a 404 when merge_request_id not found" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.